apps/condo/domains/meter/schema/_internalDeleteMeterAndMeterReadingsService.js

```diff
@@ -0,0 +1,72 @@
+/**
+ * Generated by `createservice meter._internalDeleteMeterAndMeterReadingsService --type mutations`
+ */
+
+const { isEmpty } = require('lodash')
+
+const { GQLCustomSchema, find, getById } = require('@open-condo/keystone/schema')
+
+const access = require('@condo/domains/meter/access/_internalDeleteMeterAndMeterReadingsService')
+
+const { Meter } = require('../utils/serverSchema')
+
+/**
+ * List of possible errors, that this custom schema can throw
+ * They will be rendered in documentation section in GraphiQL for this custom schema
+ */
+
+const _internalDeleteMeterAndMeterReadingsService = new GQLCustomSchema('_internalDeleteMeterAndMeterReadingsService', {
+    schemaDoc: 'Mutation to delete meters and meter readings for property',
+    types: [
+        {
+            access: true,
+            // TODO(codegen): write DeleteMeterAndMeterReadingsService input !
+            type: 'input _internalDeleteMeterAndMeterReadingsInput { dv: Int!, sender: JSON!, propertyIds: [String]! }',
+        },
+    ],
+    
+    mutations: [
+        {
+            access: access.can_internalDeleteMeterAndMeterReadings,
+            schema: '_internalDeleteMeterAndMeterReadings(data: _internalDeleteMeterAndMeterReadingsInput!): [MeterReading]',
+            resolver: async (parent, args, context) => {
+                const { data } = args
+                const { dv, sender, propertyIds } = data
+
+                const findMeters = async () => {
+                    return await find('Meter', {
+                        deletedAt: null,
+                        property: {
+                            id_in: propertyIds,
+                        },
+                    })
+                }
+
+                const meters = await findMeters()
+                if (isEmpty(meters)) {
+                    console.info('[INFO] Could not found meters by specified property ids')
+                    return
+                }
+
+                console.info(`[INFO] Following meters will be deleted: [${meters.map(reading => `'${reading.id}'`).join(', ')}]`)
+
+                const deletedMeters = await Meter.updateMany(context, meters, { dv, sender, deletedAt: 'true' })
+                let foundedMeters = []
+
+                for (const meter of deletedMeters) {
```

Comment:
usually delete mutation do not returns deleted objects
maybe, better to change return type to something, like
{ deleted: 20 } or {status:'ok'}